tests/test_types.py

```diff
@@ -1634,6 +1634,14 @@ class Model(BaseModel):
         Model(v=0.42)
 
 
+def test_strict_bytes_max_length():
+    class Model(BaseModel):
+        u: StrictBytes = Field(..., max_length=5)
+
+    with pytest.raises(ValidationError, match='byte type expected'):
+        Model(u=123)
```

Comment:
can you add a test with a byte (to make strictness pass) with
- short length to raise ValidationError
- long length to pass
?

Reply:
Done, What do you think?